src/subcommands/update.go

```diff
@@ -0,0 +1,124 @@
+package subcommands
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type UpdateArgs struct {
+	Name       string
+	IndexPath  string
+	DataType   string
+	SourceType string
+	UpdateCopy string
+	DataURI    string
```

Comment:
It seems weird to allow multiple dataURIs for add and delete but not for update

Reply:
For update we need to look up the name of the index and update the URI. I wrote it as one update per command for simplicity. Doing multiple updates in one command would mean we need multiple pairs of (index_name, updated_URI). We could do it as `semantifly update index_name1 updated_URI1 index_name2 updated_URI2 ...` but I felt it could be confusing. Should I make it as multiple updates in one command?